oat-sa/extension-tao-itemqti
# Fix: correct export assets with & in the asset path

# [TR-2929](https://oat-sa.atlassian.net/browse/TR-2929)

<General Information>

## How to test
Deployed environment Terre - http://test-tr-2929.playground.kitchen.it.taocloud.org:49831/
or
Local environment Terre - [composer.zip](https://github.com/oat-sa/extension-tao-itemqti/files/7769422/composer.zip)

1. Create Item with Graphic Interaction with a background image that has & in a title (for example [new-unsplash.zip](https://github.com/oat-sa/extension-tao-itemqti/files/7769450/new-unsplash.zip) )
2. Create a test with Item created in the first step
3. Export the test

In exported item, you should see no empty data attribute in the object
![image](https://user-images.githubusercontent.com/35266384/147245470-37ebb7fe-c1ff-45a3-8ad2-9dc885878b51.png)
